app/models/spree/payment.rb

```diff
@@ -176,8 +188,12 @@ def create_payment_profile
       gateway_error e
     end
 
+    # Inherited from Spree, makes newly entered payments invalidate previously
```

Comment:
I dont think we need to tell our story here. I'd remove "Inherited from Spree,".

Reply:
ok. What's important is the rest.